CRM/Dedupe/Merger.php

```diff
@@ -1820,7 +1820,7 @@ public static function mergeLocations($mergeHandler) {
           if (!$otherBlockId) {
             continue;
           }
-          $otherBlockDAO = $mergeHandler->copyDataToNewBlockDAO($daoName, $otherBlockId, $name, $blkCount);
+          $otherBlockDAO = $mergeHandler->copyDataToNewBlockDAO($otherBlockId, $name, $blkCount);
```

Comment:
Looks like line 1812 above (which defines the now-unused variable `$daoName`) is no longer needed.

Reply:
@colemanw - yes - removing that in this PR makes it confllct with the one in the PR template